deepsensor/model/model.py

```diff
@@ -886,190 +930,152 @@ def stitch_clipped_predictions(
-                            # to get the number of pixels to remove from left hand side of patch.
-                            if x2_ascend:
-                                prev_patch_x2_max = get_index(
-                                    patch_row_prev[var_name].coords[orig_x2_name].max(),
-                                    x1=False,
-                                )
-                                b_x2_min = (
-                                    prev_patch_x2_max - patch_x2_index[0]
-                                ) - patch_overlap[1]
-
-                            # If x2 is descending. Subtract current patch max x2 value from previous patch min x2 value to get bespoke overlap in column pixels.
-                            # To account for the clipping done to the previous patch, then subtract patch_overlap value in pixels
-                            # to get the number of pixels to remove from left hand side of patch.
-                            else:
-                                prev_patch_x2_min = get_index(
-                                    patch_row_prev[var_name].coords[orig_x2_name].min(),
-                                    x1=False,
-                                )
-                                b_x2_min = (
-                                    patch_x2_index[0] - prev_patch_x2_min
-                                ) - patch_overlap[1]
-                        else:
-                            b_x2_max = b_x2_max
-
-                        if patch_x1_index[0] == data_x1_index[0]:
-                            b_x1_min = 0
-                        # TODO: ensure this elif statement is robust to multiple patch sizes.
-                        elif abs(patch_x1_index[1] - data_x1_index[1]) < 2:
-                            b_x1_max = 0
-                            b_x1_max = b_x1_max
-                            patch_prev = preds[i - patches_per_row]
-                            if x1_ascend:
-                                prev_patch_x1_max = get_index(
-                                    patch_prev[var_name].coords[orig_x1_name].max(),
-                                    x1=True,
-                                )
-                                b_x1_min = (
-                                    prev_patch_x1_max - patch_x1_index[0]
-                                ) - patch_overlap[0]
-                            else:
-                                prev_patch_x1_min = get_index(
-                                    patch_prev[var_name].coords[orig_x1_name].min(),
-                                    x1=True,
-                                )
-
-                                b_x1_min = (
-                                    prev_patch_x1_min - patch_x1_index[0]
-                                ) - patch_overlap[0]
-                        else:
-                            b_x1_max = b_x1_max
+                first_key, first_value = next(iter(patch_pred.items()))   
```

Comment:
```suggestion
                # get one variable name to use for coordinates and extent
                first_key = list(patch_pred.keys())[0]   
```

This may just be a matter of personal preference but this seems safer and it's good practice not to assign variables that aren't needed.

Reply:
Thanks, agreed.